R/summariseObservationPeriod.R

```diff
@@ -69,27 +64,32 @@ summariseObservationPeriod <- function(observationPeriod,
       PatientProfiles::summariseResult(
         strata = "id",
         variables = c("duration", "next_obs"),
-        estimates = estimates
-      ) |>
-      suppressMessages() |>
-      dplyr::union_all(
-        obs |>
-          dplyr::group_by(.data$person_id) |>
-          dplyr::tally(name = "n") |>
-          PatientProfiles::summariseResult(
-            variables = c("n"),
-            estimates = estimates,
-            counts = F
-          ) |>
-          suppressMessages()
+        estimates = estimates[estimates != "density"]
       ) |>
```

Comment:
PatientProfiles 1.2.0 allows density to be an estimate:
to consider using that option.
To discuss

Reply:
Yes but the output is different from the one calculated with the density estimation function. I thought that was intentional